mapper/dm_agent/README.md

```diff
@@ -0,0 +1,31 @@
+# Device Monitoring
+
+Device monitoring is enabled on `thin-edge.io` using `collectd` and the `tedge-dm-agent` component.
+
+# Prerequisites
+
+Install collectd if not already installed: 
+
+`sudo apt install collectd`
+
+Configure collectd. Use the Thin Edge customized collectd configuration for basic monitoring: 
+
+`sudo cp $TEDGE_REPO/configuration/contrib/collectd/collectd.conf /etc/collectd/collectd.conf`
+
+Restart collectd after the config changes
+
+`sudo systemctl restart collectd`
+
+Validate if collectd is publishing stats to the MQTT broker
+
+`mosquitto_sub -v -t collectd/#`
```

Comment:
I think we should advice to use `tedge mqtt` and not `mosquitto_sub` especially they may not be included in `mosquitto` installation on the system.

Reply:
Yes, why don't we use tedge mqtt?

Reply:
There was no specific reason. Just habit. I'll switch to `tedge mqtt`